app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java

```diff
@@ -450,10 +450,7 @@ public void run() {
             case R.id.button_ok:
 
                 Bundle result = new Bundle();
-                result.putString(
-                        EXTRA_KEY_AREA,
-                        Double.toString(areaCalculator.getArea())
-                );
+
                 result.putString(
                         EXTRA_KEY_PERIMETER,
```

Comment:
The callout needs to have 1 return value - either area or perimeter. The other can be returned as an optional extra. 

I assumed that area would be the default return value...but I could change that, or make it configurable.